odata-csdl/14 Vocabulary and Annotation.md

```diff
@@ -1204,26 +1204,25 @@ Example ##ex:
 ## ##subsec Dynamic Expression
 
 Dynamic expressions allow assigning a calculated value to an applied
-term.
+term or term property.
 
 ### ##subsubsec Path Expressions
 
 Path expressions allow assigning a value to an applied term or term
 component. There are two kinds of path expressions:
-- A *model path* is used within
+- A *model path* is evaluated on the entity model of a service and resolves to a model element.
+Model paths are used within
 [Annotation Path](#AnnotationPath), [Model Element Path](#ModelElementPath),
-[Navigation Property Path](#NavigationPropertyPath), and [Property
-Path](#PropertyPath) expressions to traverse the model of a service and
-resolves to the model element identified by the path. It allows
-assigning values to terms or term properties of the [built-in
-types](#BuiltInTypesfordefiningVocabularyTerms) `Edm.AnnotationPath`,
-`Edm.NavigationPropertyPath`, `Edm.PropertyPath`, and their base types
-`Edm.AnyPropertyPath` and `Edm.ModelElementPath`.
-- An *instance path* is used within a
-[Value Path](#ValuePath) expression to traverse a graph of type
-instances and resolves to the value identified by the path. It allows
-assigning values to terms or term properties of built-in types other
-than the `Edm.*Path` types, or of any model-defined type.
+[Navigation Property Path](#NavigationPropertyPath), and [Property Path](#PropertyPath) expressions.
+They allow assigning values to terms or term properties of the 
+[built-in types](#BuiltInTypesfordefiningVocabularyTerms) `Edm.AnnotationPath`,
+`Edm.ModelElementPath`, `Edm.NavigationPropertyPath`, `Edm.PropertyPath`, and
+`Edm.AnyPropertyPath`.
+- An *instance path* is evaluated on a graph of type instances and resolves to the
```

Comment:
The word "graph" appears only twice in the text, but is not really defined. Can we avoid it?

Reply:
How about "directed graph of type instances"? That's a well-known mathematical term.

Reply:
Is it always a single "start" instance, or can that already be a collection of type instances?

Reply:
The only expressions that are evaluable on a collection are the ones introduced in OData-Data-Agg-v4.0, section 3.6. For instance paths, there is always a single start instance (although that may be a "loop variable", for example, with `$filter` or `$compute`).